packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -103,9 +93,10 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
 		logger: ITelemetryBaseLogger,
 		summarizeInternalFn: SummarizeInternalFn,
 		config: ISummarizerNodeConfigWithGC,
+		_summaryHandleId: EscapedPath,
 		changeSequenceNumber: number,
-		/** Undefined means created without summary */
-		latestSummary?: SummaryNode,
+		/** Summary reference sequence number, i.e. last sequence number seen when it was created */
+		_lastSummaryReferenceSequenceNumber?: number,
```

Comment:
nit: don't use underscore since this class doesn't do anything with these two properties
```suggestion
		lastSummaryReferenceSequenceNumber?: number,
```